src/main/java/org/sonarlint/intellij/core/BackendService.kt

```diff
@@ -315,7 +315,9 @@ class BackendService : Disposable {
         val omnisharpRequirementsDto = OmnisharpRequirementsDto(
             getService(SonarLintPlugin::class.java).path.resolve("omnisharp/mono"),
             getService(SonarLintPlugin::class.java).path.resolve("omnisharp/net6"),
-            getService(SonarLintPlugin::class.java).path.resolve("omnisharp/net472")
+            getService(SonarLintPlugin::class.java).path.resolve("omnisharp/net472"),
```

Comment:
Nitpick: `getService(SonarLintPlugin::class.java).path` could be extracted to a variable (used 5 times).